### PR TITLE
Add wordTokenizer script

### PR DESCRIPTION
**Pull request by erezh31**

### CODE REVIEW

File: Scripts/WordTokenizerV2/WordTokenizer.py [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+
+
+def word_tokenize(text):
+    if VALUE_IS_JSON:
+        try:
+            text = json.loads(text)
+        except:
+            pass
+
+    if type(text) is not list:
```
Comment (Itay4) [inline Scripts/WordTokenizerV2/WordTokenizer.py]: use `isinstance`

File: Scripts/WordTokenizerV2/WordTokenizer.py [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+        text = [text]
+    result = map(remove_line_breaks, map(tokenize_text, map(clean_html, text)))
+
+    if len(result) == 1:
+        result = result[0]
+
+    demisto.results({
```
Comment (Itay4) [inline Scripts/WordTokenizerV2/WordTokenizer.py]: use `return_outputs`

File: Scripts/WordTokenizerV2/WordTokenizer.py [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+
+    return re.sub(r"\s+", " ", text.replace("\r"," ").replace("\n"," ")).strip()
+
+
+def word_tokenize(text):
+    if VALUE_IS_JSON:
+        try:
+            text = json.loads(text)
+        except:
+            pass
```
Comment (Itay4) [inline Scripts/WordTokenizerV2/WordTokenizer.py]: are you sure you dont want to handle text somehow differently here?

File: Scripts/WordTokenizerV2/WordTokenizer.yml [hunk 4]
```diff
@@ -0,0 +1,268 @@
...
+  name: removeStopWords
+  predefined:
+  - 'yes'
+  - 'no'
+  required: false
+  secret: false
+- auto: PREDEFINED
+  default: false
+  defaultValue: 'yes'
+  description: 'Remove punctuation '
```
Comment (Itay4) [inline Scripts/WordTokenizerV2/WordTokenizer.yml]: remove white space

File: Scripts/WordTokenizerV2/WordTokenizer.py [hunk 5]
```diff
@@ -0,0 +1,104 @@
+import demistomock as demisto
+from CommonServerPython import formats, hash_djb2
+import spacy, re, json
+from HTMLParser import HTMLParser
+
+CLEAN_HTML = (demisto.args()['cleanHtml'] == 'yes')
+REMOVE_LINE_BREAKS = (demisto.args()['removeLineBreaks'] == 'yes')
+TEXT_ENCODE = demisto.args()['encoding']
```
Comment (Itay4) [inline Scripts/WordTokenizerV2/WordTokenizer.py]: this is not used

File: Scripts/WordTokenizerV2/WordTokenizer.py [hunk 6]
```diff
@@ -0,0 +1,104 @@
...
+        return text
+
+    cleaned = text
+    for pattern in HTML_PATTERNS:
+        cleaned = pattern.sub(" ", cleaned)
+    return html_parser.unescape(cleaned).strip()
+
+
+def tokenize_text(text):
+    doc = nlp(unicode(text))
```
Comment (Itay4) [inline Scripts/WordTokenizerV2/WordTokenizer.py]: might wanna wrap unicode in try-except

**Comment by erezh31** [comment]
@Itay4 fixed CR

**Comment by Itay4** [comment]
@erezh31 looks good, approving on my side, but please make sure @kirbles19 reviews the yml

**Pull request merged**